packages/codemods/DOCUMENTATION.md

```diff
@@ -0,0 +1,511 @@
+# HPE Design System Codemods Documentation
+
+## Overview
+
+`hpe-design-system-codemods` is a codemod project to assist migrations for package versions related to the HPE Design System.
+
+Available codemod transformations:
+
+- [Grommet-theme-hpe v6 to v7](#Grommet-theme-hpe-v6-to-v7)
+
+## Table of Contents
+
+- [Installation](#installation)
+- [Transforms](#transforms)
+- [Quick Start](#quick-start)
```

Comment:
Keep ahead of Transforms and make the the quick start content more generic?

Reply:
Done